app/templates/components/events/event-import-section.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="ui stackable centered grid">
+  <div class="sixteen wide column">
+    <h3 class="ui header">{{t 'Import Event'}}</h3>
+    <div class="ui divider"></div>
+    <p class="section-heading">{{t 'Select event source file to import'}}</p>
+    <p><b>{{t 'Supported formats:'}}</b> {{t 'Open Event compatible json package (.zip), PentabarfXML (.xml), iCalendar (.ical)'}}</p>
```

Comment:
Keep the colon out of the translation helper. We don't want that to go missing due to some translator.